test/extended/networking.sh

```diff
@@ -85,50 +99,81 @@ function save-artifacts() {
   copy-container-files "/etc/hosts" "${dest_dir}"
 }
 
-# Any non-zero exit code from any test run invoked by this script
-# should increment TEST_FAILURE so the total count of failed test runs
-# can be returned as the exit code.
-TEST_FAILURES=0
-function test-osdn-plugin() {
+function deploy-cluster() {
   local name=$1
   local plugin=$2
   local isolation=$3
+  local log_dir=$4
```

Comment:
Unclear if it's needed, but some sort of `if [[ $# -lt 4 ]]; then os::log::error "deploy-cluster expected four arguments, got $#"; fi` statement might be useful as a guard here as we have `set -o nounset` on.